### PR TITLE
Wip/wikidata single turn

### PR DESCRIPTION
**Pull request by rayslxu**

### CODE REVIEW

File: tool/tokenize.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        process.nextTick(callback);
+    }
+}
+
+module.exports = {
+    initArgparse(subparsers) {
+        const parser = subparsers.add_parser('tokenize', {
+            add_help: true,
+            description: "Preprocess (tokenize) a string value dataset."
+        });
```
Comment (gcampax) [inline tool/tokenize.js:73]: This script is redundant with preprocess-string-dataset, right?

**Review by gcampax — APPROVED** [review]
Overall, looks good. Do we want to add the wikidata starter as a test in Travis? So it won't regress in the future.

File: tool/tokenize.js (hunk 1 above)
Comment (rayslxu) [reply tool/tokenize.js:73]: oh, sorry. this was a script I used to re-tokenize training examples... it sneaked in by accident.

**Review by gcampax — APPROVED** [review]
Still yes

**Pull request merged**